pkg/tracingpolicy/examples/tracepoint-lseek-pid.yaml

```diff
@@ -8,7 +8,9 @@ spec:
     event: "sys_enter_lseek"
     args:
     - index: 7
+      type: "auto"
```

Comment:
"auto" is the default, so this is technically not needed, but doesn't hurt either

Reply:
got it 👍😊